src/server/server.js

```diff
@@ -58,10 +61,14 @@ if (!PROD) {
   app.use(require('webpack-hot-middleware')(compiler)); // eslint-disable-line import/no-extraneous-dependencies
   /* eslint-enable global-require */
 }
-
+Raven.config(process.env.SENTRY_DSN, {
+  release: version,
+  environment: process.env.NODE_ENV,
+  parseUser: jwtFields
+}).install();
 // setup middleware
 // sentry.io request handler capture middleware, must be first:
-app.use(raven.requestHandler(process.env.SENTRY_DSN));
+app.use(Raven.requestHandler());
```

Comment:
Do we need to conditionally not use this middleware if SENTRY_DSN is not set to re-enable server console logs?

Reply:
server logs are based off the the NODE_ENV, not the presence of sentry

Reply:
Hmm, that isn't the behavior I was observing when I ran my test using `yarn dev`.

Reply:
ah, i think i know what you were talking about, where sentry still sends a warning to the console? i fixed that in #1869